src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -935,6 +935,10 @@ void program_node::init_onednn_primitive_attributes() {
     // Added this for debug purposes only
     size_t empty_mem = 0xff;
 
+    // Change scratchpad mode to user
+    if (attrs->get_scratchpad_mode() == dnnl::scratchpad_mode::library)
+        attrs->set_scratchpad_mode(dnnl::scratchpad_mode::user);
```

Comment:
Can it be done unconditionally?

Reply:
No problem. Because there is no change if onednn primitive has no scratchpad.

Reply:
I think code was not changed after comment. could you please follow-up?

Reply:
Sorry for the confusion. I said that there is no need for conditions. Using user mode has no effect on onednn primitives, which do not use scratchpad anyway. Are there any plans to support library mode?

Reply:
I think what Vladimir meant is to remove this condition `    if (attrs->get_scratchpad_mode() == dnnl::scratchpad_mode::library)`
But I think this is not a big deal at this moment.